src/richie/apps/search/viewsets/courses.py

```diff
@@ -2,14 +2,17 @@
 API endpoints to access courses through ElasticSearch
 """
 from django.conf import settings
+from django.utils.module_loading import import_string
 from django.utils.translation import get_language_from_request
 
 from elasticsearch.exceptions import NotFoundError
 from rest_framework.response import Response
 from rest_framework.viewsets import ViewSet
 
 from ..forms import CourseListForm
-from ..indexers.courses import CoursesIndexer
+
+# Get the courses indexer from the settings
+indexer = import_string(settings.ES_INDICES.courses)
```

Comment:
I think naming it `CoursesIndexer` was more explicit; why did you change it to a generic name?

Reply:
I can name it `courses_indexer` but not `CoursesIndexer` :)

Reply:
:sweat: True. Let's do this.